index.js

```diff
@@ -78,7 +79,7 @@ Deployer.prototype.deploy = function(version, imagePath) {
 
       var deferreds = [];
       for (var i = 0; i < d.services.length; i++) {
-        deferreds.push(d.services[i].deploy(version, imagePath));
+        deferreds.push(d.services[i].deploy(version, d.services[i].service.imagePath));
```

Comment:
The `imagePath` variable will now be accessed from the manifest instead of from the function call.
Original behavior is maintained if the manifest doesn't include `imagePath` (would be carried through as 'undefined' in this case).